### PR TITLE
Add Pulp 3 HTTP URLs

### PR DESCRIPTION
**Pull request by ekohl**
This matches the pulp3.yml.example file with defaults that match the theforeman-pulpcore defaults.

### CODE REVIEW

File: manifests/plugin/pulp.pp [hunk 1]
```diff
@@ -25,6 +25,10 @@
 #
 # $pulp3_mirror::       Whether this pulp3 plugin acts as a mirror or another pulp node. A pulp3 mirror is the pulp3 equivalent of a pulpnode.
 #
+# $pulp3_api_url::      The URL to the Pulp 3 API
+#
+# $pulp3_content_url::  The URL to the Pulp 3 content
```
Comment (ehelms) [inline manifests/plugin/pulp.pp]: We've already used this convention in a few places, but in others we've identified that the use of `pulp3` may come back to bite us and used `pulpcore`.

Comment (ekohl) [reply manifests/plugin/pulp.pp]: I don't mind renaming. Most of these predate the pulpcore convention. In https://github.com/theforeman/smart_proxy_pulp/ it's also the pulp3 module.

Comment (ehelms) [reply manifests/plugin/pulp.pp]: Let's merge this and I'll raise an issue here to discuss further.

**Comment by ekohl** [comment]
@jlsherrill these defaults make sense, right? Or should we default to the public URLs?

**Comment by ekohl** [comment]
Updated to the public URLs by default to match how we deploy this.

**Comment by ekohl** [comment]
Updated to the public URLs by default to match how we deploy this.

**Pull request merged**